andela/team-odd-bn-frontend
# #170574394 user should be able to verify email address

#### What does this PR do?
This PR adds email verification page after user signs up
#### Description of Task to be completed?
- A user is able to resend a verification email after signup
- A user is able to navigate directly to the app after email verification
- User is able to get a successful registration page
#### How should this be manually tested?
- `git clone https://github.com/andela/team-odd-bn-frontend.git`
- `cd team-odd-bn-frontend`
- `git checkout ft-user-verification-email-170574394`
- `npm install` packages
- `npm run dev` to start the development server
- It should open your default browser automatically
- Navigate to `localhost:4000/verify-email` to access the page directly
#### Any background context you want to provide?
Previously, the users used to validate their email using the backend part
#### What are the relevant pivotal tracker stories?
[#170574394](https://www.pivotaltracker.com/story/show/170574394)
#### Screenshots (if appropriate)
###### verify email page
<img width="1440" alt="Screen Shot 2020-01-15 at 09 35 46" src="https://user-images.githubusercontent.com/35301129/72414261-7cf21880-377a-11ea-813e-43b430baf5b2.png">

###### Successful email verication page
<img width="1440" alt="Screen Shot 2020-01-10 at 02 01 13" src="https://user-images.githubusercontent.com/35301129/72114743-24231a00-334d-11ea-8178-42f277a94ab8.png">

#### Questions:
N/A
